theon/src/x86_64/multiboot1.rs

```diff
@@ -5,6 +5,7 @@
 // license that can be found in the LICENSE file or at
 // https://opensource.org/licenses/MIT.
 
+use crate::end_addr;
```

Comment:
Can you bust this out into another commit?

Reply:
Done.

Reply:
Done.